assets/js/hooks/Mapper/components/mapInterface/widgets/SystemSignatures/constants.ts

```
@@ -208,3 +223,51 @@ export const SIGNATURE_DELETION_TIMEOUTS: SignatureDeletionTimingType = {
+    [SignatureKindDE.Structure]: SignatureKind.Structure,
+    [SignatureKindDE.Ship]: SignatureKind.Ship,
+    [SignatureKindDE.Deployable]: SignatureKind.Deployable,
+    [SignatureKindDE.Drone]: SignatureKind.Drone,
+    [SignatureKindDE.Starbase]: SignatureKind.Starbase,
+  },
+};
+
+// Flatten the structure for backward compatibility
+export const MAPPING_TYPE_TO_ENG: Record<string, SignatureKind> = Object.entries(LANGUAGE_TYPE_MAPPINGS).reduce(
+  (result, [, mappings]) => {
+    return { ...result, ...mappings };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)